### DISCUSSION TITLE
Conftest files in CodeQL Code scanning.

### DISCUSSION THREAD

**Discussion by dlemstra** [github/securitylab#125] [General] [answered]
We are using the GitHub code scanning beta feature in our @ImageMagick repository. The @ImageMagick library is a C/C++ library and it uses `./configure` to configure the library. During the configuration stage a couple of `conftest.c` files are generated to check if a method is available. And this gets reported as an issue:

![afbeelding](https://user-images.githubusercontent.com/10426229/84565551-3412b000-ad6a-11ea-8ce5-ca82837f091f.png)

 At this point in time I am not sure if we have a mistake in our configure script or that this is by design. As you can see the `conftest.c` file is not available because this file is created and then cleaned up afterwards. The content of the file `config.log` could be used to track down the method that is called incorrectly. I tried to go through it myself but I was unable to find the method that is being called incorrectly. But there might also be a much better way to get the contents of the file that is causing this issue. This is probably also an issues for other files that are generated during the build and removed afterwards.

**Comment by jhutchings1** [comment, marked as answer]
Thanks @dlemstra! This is a known issue on our side. We don't currently render snippets from files that are generated during the build. We've got an item on our backlog to address this in the future.
